app/assets/stylesheets/users.css

```diff
@@ -0,0 +1,76 @@
+/* index.html.erb */
+.users-list {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+  list-style-type: none;
+  gap: 10px;
+  padding: 0;
+}
+
+.users-list-item {
+  width: 100%;
+}
+
+.user-partial {
+  display: flex;
+  width: 100%;
+  position: relative;
+  border: 1px solid gray;
+}
+
+.user-partial > img {
+  width: 150px;
+  margin-right: 10px;
+}
+
+.num-of-posts {
+  position: absolute;
+  bottom: 0;
+  right: 20px;
+}
+
+/* show.html.erb */
+.user {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  gap: 10px;
+}
+
+.user-bio {
+  border: 1px solid gray;
+  padding: 0 10px;
+  margin-bottom: 20px;
+}
+
+.user-posts-preview {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
```

Comment:
- [ ] Awesome job! Your views are working great! However, let's add a few tweaks to ensure that the design complies with the wireframe provided; Let's ensure that the user picture is not joined in one border with the user bio.

**Your designs:**
**View 1**        |  **View 2**        |  **View 3**
:-------------------------:|:-------------------------:|:-------------------------:
<img width="800" alt="Your desgin" src="https://github.com/shasherazi/blog/assets/12745474/82295f7e-35ad-4789-a39e-b2f34c5eee2f"> |  <img width="800" alt="Home Page" src="https://github.com/shasherazi/blog/assets/12745474/272b55be-c575-409d-99ad-302895f4d3a4"> | <img width="800" alt="Home Page" src="https://github.com/shasherazi/blog/assets/12745474/23fcd807-5880-411b-8102-34bc8f17dab7">

**Wireframe designs:**
**View 1**        |  **View 2**        |  **View 3**
:-------------------------:|:-------------------------:|:-------------------------:
<img width="800" alt="Your desgin" src="https://github.com/shasherazi/blog/assets/12745474/a9461f3b-2109-4ba6-ac56-078df13764d3"> |  <img width="800" alt="Home Page" src="https://github.com/shasherazi/blog/assets/12745474/3f7d0f65-2c5d-422d-a93c-92b8ab262892"> | <img width="800" alt="Home Page" src="https://github.com/shasherazi/blog/assets/12745474/ff73b457-3699-46e0-9e40-a0de68408c27">

Reply:
Fixed.